airflow/ti_deps/ti_deps.py

```diff
@@ -0,0 +1,479 @@
+from abc import ABCMeta, abstractmethod
```

Comment:
I don't feel strongly about it, my main concern is the py2/py3 issue and the fact that it's not a commonly used package